agent/build.gradle

```diff
@@ -50,26 +50,28 @@ project.tasks.jarFileTest.configure {
 tasks.check.dependsOn tasks.jarFileTest
 
 dependencies {
-    compileOnly 'com.google.auto.service:auto-service:1.0-rc7'
+    compileOnly 'com.google.auto.service:auto-service-annotations:1.0-rc7'
+    annotationProcessor 'com.google.auto.service:auto-service:1.0-rc7'
 
-    compile 'net.bytebuddy:byte-buddy:1.10.19'
-    compile 'net.bytebuddy:byte-buddy-agent:1.10.19'
+    implementation 'net.bytebuddy:byte-buddy:1.10.19'
+    implementation 'net.bytebuddy:byte-buddy-agent:1.10.19'
+    //FIXME should we activate byte-buddy's annotation processing here?
```

Comment:
@bsideup as far as I can see, we don't need to add the `annotationProcessor` dependency (ie we don't _use_ ByteBuddy's annotation processing in BlockHound's code), is that correct?

note that since Gradle 5 annotations processors are not added by `compile` (see https://docs.gradle.org/current/userguide/upgrading_version_4.html#rel4.6:annotation_processor_configuration)

Reply:
I am not aware of any annotation processing in ByteBuddy.